app/src/main/res/values/strings.xml

```diff
@@ -9,8 +9,12 @@
     <string name="temperature_unit">Temperature Unit</string>
     <string name="wind_speed_unit">Wind speed Unit</string>
     <string name="datetime_format">dd/MM/yyyy HH:mm:ss</string>
+    <string name="permission_denied_explanation">Permission was denied, but is needed for core
```

Comment:
WARNING: [UnusedResources] The resource `R.string.permission_denied_explanation` appears to be unused
 Unused resources make applications larger and slow down builds.